### PR TITLE
ASC-1627 Add test for cinder service

### PR DESCRIPTION
**Pull request by johnduarte**

### CODE REVIEW

File: molecule/default/tests/test_openstack_services.py [hunk 1]
```diff
@@ -38,6 +46,42 @@ def service_enabled(service, run_on_host):
     else:
         return False
 
+
+def get_openstack_release(run_on_host):
```
Comment (phongdly) [inline molecule/default/tests/test_openstack_services.py]: This function will be used in future tests, wondering if you could put it in other place for universal access like a fixture in conftest.py or in pytest-rpc?

**Review by phongdly — CHANGES_REQUESTED** [review]

File: molecule/default/tests/test_openstack_services.py (hunk 1 above)
Comment (ryan-rs) [reply molecule/default/tests/test_openstack_services.py]: Definitely could be ported to pytest-rpc.

**Review by ryan-rs — APPROVED** [review]

**Comment by johnduarte** [comment]
This PR is blocked by the merge of the `get_cinder_major_version` helper into pytest-rpc: https://github.com/rcbops/pytest-rpc/pull/65 and the subsequent release of a new version of pytest-rpc.

**Comment by johnduarte** [comment]
Version 1.1.0 of pytest-rpc has been released with the `get_cinder_major_version` helper.  This PR is no longer blocked.
@phongdly please take another look at this at your convenience.

**Review by phongdly — APPROVED** [review]

**Pull request merged**